### PR TITLE
Feature/gdev96

### PR DESCRIPTION
**Pull request by KerstenBreuer**
Added examples for the "models" module and the "core" sub-package.

Please note, (as #3) this PR depends on the shared lib at https://github.com/ghga-de/ghga-service-chassis-lib/tree/feature/rest which is not yet available at PyPI. Thus to test it now run the following:

pip install git+https://github.com/ghga-de/ghga-service-chassis-lib@feature/rest
dev_install
This is also why the CI currently fails. Once the shared library is realized, the installation will work as intended.

This PR depends on #1 and #3, please review them first.

### CODE REVIEW

File: my_microservice/core/greeting.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+"""This module contains functionalities for greeting persons."""
+
+from datetime import datetime
+import random
+from ..models import GreetingExpression, Greeting
+
+GREETINGS_EXPRESSIONS = [
+    GreetingExpression(expression="Kaliméra", language="Greek", isinformal=False),
+    GreetingExpression(expression="Yassou", language="Greek", isinformal=True),
```
Comment (ac-jorellanaf) [inline my_microservice/core/greeting.py:24]: This hurts me, φίλε.

**Review by ac-jorellanaf — APPROVED** [review]

File: .devcontainer/.my-microservice.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Please adapt to package name:
+# rename the file name to match the
+# "prefix" configured in the config
+# module
+host: 127.0.0.1
+port: 8080
+log_level: info
+workers: 1
+auto_reload: true
+language: Greek
```
Comment (ac-jorellanaf) [inline .devcontainer/.my-microservice.yaml:10]: The comment I had written got deleted, but I will reiterate. I take it you are hardcoding using Greek for the greeting language generation when you do an API call with this default config? I also imagine you can call the method and send this parameter with a different value?

File: my_microservice/core/greeting.py (hunk 1 above)
Comment (ac-jorellanaf) [reply my_microservice/core/greeting.py:24]: Also consider Γεια σας (Geia sas/Yassas*) as another formal greeting 😉

**Review by lkuchenb — APPROVED** [review]
👍

File: my_microservice/core/greeting.py (hunk 1 above)
Comment (KerstenBreuer) [reply my_microservice/core/greeting.py:24]: Haha, if you like please feel free to add it the correct spelling 😄

File: .devcontainer/.my-microservice.yaml (hunk 2 above)
Comment (KerstenBreuer) [reply .devcontainer/.my-microservice.yaml:10]: Yes exactly that would be possible, but it's just a toy example, so let's not put too much effort into it 😄

**Pull request merged**